src/inverseRobot.js

```diff
@@ -23,8 +23,53 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  if (!robot) {
+    return {};
+  }
+
+  const keys = Object.keys(robot).reverse();
+  const values = Object.values(robot).reverse();
+  const tem = {};
```

Comment:
what is tem? use proper names for variables

Reply:
that was temporary in my minds.. changed already